source/_posts/2024-11-27-home-assistant-yellow-gets-cm5-support.markdown

```diff
@@ -0,0 +1,67 @@
+
+## Installing on CM5
+
+For the Home Assistant Yellow, we have [two ways to install Home Assistant OS](https://yellow.home-assistant.io/guides/reinstall-os/) onto the Compute Module. One is very easy and quick (using USB 2.0), while the other is more complex (using rpiboot). Unfortunately, due to firmware differences with CM5 it cannot boot off USB 2.0 devices (though the USB 2.0 ports work once the device is booted).
+
+If you already have a Yellow running Home Assistant OS, upgrading to CM5 can be a drop-in replacement, but in some circumstances it can be more complex,
+
+- **CM4 Lite (no eMMC) with NVMe storage** - Update to the latest HAOS (version 14.0 or greater - as of writing 14.0 is still pre-release, but will be stable when CM5s start to reach consumers), power it down, swap the CM4 Lite for a CM5 Lite, and you're good to go.
+
+- **CM4 with eMMC (regardless if you are using NVMe or not)** -  Download a backup of your Home Assistant, power down your system, and install Home Assistant OS on the CM5 using [rpiboot](https://yellow.home-assistant.io/guides/reinstall-os/#:~:text=Option%202%3A%20Reinstall%20Home%20Assistant%20OS%20using%20rpiboot) (the more complex installation method). Once installed restore the backup.
+
+- **New Yellow with CM5**: You will need to install Home Assistant with [rpiboot](https://yellow.home-assistant.io/guides/reinstall-os/#:~:text=Option%202%3A%20Reinstall%20Home%20Assistant%20OS%20using%20rpiboot) (the more complex installation method).
+
+For full details on how to set up your Home Assistant Yellow [visit our documentation](https://yellow.home-assistant.io/).
+
+## CM4 is still great
+
+If you have, or were looking at getting, a Home Assistant [Green](/green/) or [Yellow](/yellow/) with a CM4, both are more than capable. A third of all Home Assistant users are using Pi 4 class hardware.
+
+<img src='/images/blog/2024-11-cm5/analytics.png' style='border: 0;box-shadow: none;' alt="In our analytics Pi 4 class hardware is the most used SBC">
+
+In fact, according to our [opt-in analytics](https://analytics.home-assistant.io/), more people are using Pi 3 hardware than Pi 5 hardware (granted the Pi 5 has only had official support since [February this year](/blog/2024/02/26/home-assistant-os-12-support-for-raspberry-pi-5/)).  Home Assistant continues to get updates that improve its speed on all hardware, most recently [faster backups](/blog/2024/02/26/home-assistant-os-12-support-for-raspberry-pi-5/#faster-backups) and [reboots](/blog/2024/03/06/release-20243/#home-assistant-boots-twice-as-fast). This allows people to keep hardware running longer, and in the end that is our goal 🌎.
```

Comment:
```suggestion
In fact, according to our [opt-in analytics](https://analytics.home-assistant.io/), more people are using Pi 3 hardware than Pi 5 hardware (granted the Pi 5 has only had official support since [February this year](/blog/2024/02/26/home-assistant-os-12-support-for-raspberry-pi-5/)). Home Assistant continues to get updates that improve its speed on all hardware, most recently [faster backups](/blog/2024/02/26/home-assistant-os-12-support-for-raspberry-pi-5/#faster-backups) and [reboots](/blog/2024/03/06/release-20243/#home-assistant-boots-twice-as-fast). This allows people to keep hardware running longer, and in the end that is our goal 🌎.
```